src/app.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <HelloWorld msg="Welcome to Vue 3 App" />
+</template>
+
+<script lang="ts">
+import { Options, Vue } from "vue-class-component";
+import HelloWorld from "./components/hello-world/index.vue";
```

Comment:
We should rename `HelloWorld` component to `label` component. That seem more generic.

We should also not pass message as property. We should be able to write following markup

```
<label>Hello World</label>
```